packages/systemverilog/systemverilog.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "Xavier Guérin <ghub@applepine.org>"
+authors: ["Xavier Guérin <ghub@applepine.org>"]
+homepage: "https://github.com/xguerin/systemverilog"
+doc: "https://xguerin.github.io/systemverilog/wiki"
+license: "ISC"
+dev-repo: "https://github.com/xguerin/systemverilog.git"
+bug-reports: "https://github.com/xguerin/systemverilog/issues"
+tags: []
+available: [ ocaml-version >= "4.01.0"]
+substs: [ "pkg/META" ]
+depends:
+[
+  "ocamlfind"  { build                 }
+  "ocamlbuild" { build                 }
+  "topkg"      { build & >= "0.9.0"    }
+  "astring"    { build & >= "0.8.3"    }
```

Comment:
cool thanks, I was just checking (it's a bit unusual to have `astring` as a build dep)